packages/mdc-select/foundation.ts

```diff
@@ -96,6 +98,7 @@ export class MDCSelectFoundation extends MDCFoundation<MDCSelectAdapter> {
     this.helperText_ = foundationMap.helperText;
 
     this.menuItemValues_ = this.adapter_.getMenuItemValues();
+    this.disabled_ = false;
```

Comment:
We can initialize assign default value (`false`) during initialization of this variable.